charts/outline/templates/001-deployment.yaml

```diff
@@ -32,6 +32,8 @@ spec:
           volumeMounts:
           - name: {{ .Release.Name }}-data
             mountPath: {{ .Values.outline.fileStorage.localRootDir }}
+          securityContext:
+            runAsUser: 0
```

Comment:
It would be better if we make this optional:

```
{{- if .Values.outline.fileStorage.initContainerSecurityContext.enabled }}
    securityContext:
         runAsUser: 0
{{- end }}
```

This will require adding the following lines in `values.yaml`:

`## @param outline.fileStorage.initContainerSecurityContext.enabled Whether to set the security context for the initContainer. Useful for deployments on GKE and similar.`,

under 

`## @param outline.fileStorage.uploadMaxSize Maximum file upload size limit.`

and adding the value itself:

```
outline:
  fileStorage:
    type: local # set to local|s3
    .
    .
    .
    initContainerSecurityContext:
      enabled: false
```

And don't forget to run:

`readme-generator -v charts/outline/values.yaml -r charts/outline/README.md`

before commiting.